Source/miniwin/miniwin.h

```diff
@@ -2,8 +2,22 @@
 
 #include <stdint.h>
 
+#include <ctype.h>
+#include <math.h>
+#include "utils/stdarg_compat.h"
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#include "storm/storm_full.h"
+
 namespace devilution {
 
+#ifndef MAX_PATH
+#define MAX_PATH 260
+#endif

```

Comment:
?

Reply:
The max length of a file path as defined in WinAPI on Windows 95. This may be already defined on Windows platforms and should only be conditionally defined.

Again this comes from the merging of `SourceS/miniwin/misc.h` and `SourceS/miniwin.h`.

Hope that answers you questionmark.